account_move_batch_validate/wizard/move_marker.py

```diff
@@ -38,11 +38,16 @@ class AccountMoveMarker(orm.TransientModel):
             ('mark', 'Mark for posting'),
             ('unmark', 'Unmark for posting'),
         ], "Action", required=True),
-        'eta': fields.integer('Seconds to wait before starting the jobs')
+        'eta': fields.integer('Seconds to wait before starting the jobs'),
+        'journal_ids': fields.many2many('account.journal',
+                                        'account_post_journal_rel',
+                                        'wiz_id', 'journal_id', 'Journals',
+                                        required=True),
```

Comment:
Well, it's a matter of taste, but I prefer to not having anything, and add only the desired ones when needed, than removing everything except one, that it's the common task. I mean, my usual tasks are:

* Validate all journals.
* Validate only 1 journal.

In these use cases, I prefer my approach.